gordo/builder/mlflow_utils.py

```diff
@@ -435,11 +435,11 @@ def log_machine(mlflow_client, run_id, machine: Machine):
 
     # Send configs as JSON artifacts
     try:
-        with tempfile.TemporaryDirectory("./") as tmp_dir:
```

Comment:
I dont understand :-p Was the `"./"` interpreted as the `mode` parameter to ` TemporaryDirectory` ?

Reply:
Think it was treated as the suffix, which then effectively did nothing.

Reply:
I think it broke after the non-root user changes on the docker image, at which point it didn't have access to write to /tmp